docs/out_of_core.rst

```diff
@@ -1,46 +1,28 @@
-Out of Core in Modin (experimental)
-===================================
+Out of Core in Modin
+====================
 
 If you are working with very large files or would like to exceed your memory, you may
 change the primary location of the `DataFrame`_. If you would like to exceed memory, you
-can use your disk as an overflow for the memory. This API is experimental in the context
-of Modin. Please let us know what you think!
-
-Install Modin out of core
--------------------------
-
-Modin now comes with all the dependencies for out of core functionality by default! See
-the :doc:`installation page </installation>` for more information on installing Modin.
+can use your disk as an overflow for the memory.
 
 Starting Modin with out of core enabled
 ---------------------------------------
 
-Out of core is detected from an environment variable set in bash.
-
-.. code-block:: bash
-
-   export MODIN_OUT_OF_CORE=true
-
-We also set up a way to tell Modin how much memory you'd like to use. Currently, this
-only accepts the number of bytes. This can only exceed your memory if you have enabled
-``MODIN_OUT_OF_CORE``.
+Out of core is now enabled by default for both Ray and Dask engines.
```

Comment:
Should we add version notes for Ray and Dask here? Should we bump the requirements?

Reply:
I've also noticed that it _might_ start working starting with Ray 1.3.0 which isn't yet released.
Am I being wrong here?